### ISSUE TITLE
[RFC]: add support for exporting API docs to a PDF

### ISSUE THREAD

**Issue by kgryte** [stdlib-js/www#30]
### Description

This RFC proposes adding support for exporting API docs to a PDF.

PDF documentation could be useful outside of the website and in the context of offline usage. PDF docs are common in, e.g., R and TeX documentation.

Generation of the PDF could be done either on-demand (client side; see [htm2pdf.js](https://ekoopmans.github.io/html2pdf.js/)) or pre-compiled on the server and simply sent on request. 

HTML to PDF can be problematic, as a common technique is to capture HTML as an image, which prevents the generated PDF text from being searchable or selectable.

This RFC would benefit from [gh-27](https://github.com/stdlib-js/www/issues/27), where JSON could be used to automatically generate TeX which could then be compiled to a PDF.

### Related Issues

Related issues #27.

### Questions

No.

### Other

No.

### Checklist

- [X] I have read and understood the [Code of Conduct](https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md).
- [X] Searched for existing issues and pull requests.
- [X] The issue name begins with `RFC:`.

**Comment by Planeshifter** [comment]
Alternatively, could use [https://github.com/bpampuch/pdfmake](pdfmake) in combination with [html-to-pdfmake](https://github.com/Aymkdn/html-to-pdfmake), which I have been using with some success.

**Comment by kgryte** [comment]
Yeah, my sense is that anything HTML-to-PDF will be tricky/imperfect given our heavy reliance on CSS, and that a more robust conversion pipeline over the long term would be JSON-to-TeX.

**Comment by Planeshifter** [comment]
While not getting us near the ability to export the entire docs as a PDF document, I tweaked our CSS a bit per https://github.com/stdlib-js/www/commit/8eec861b0fce7171b814d9132474c3838679879c to allow one to nicely print a single README.md without any interface element.

**Comment by Planeshifter** [comment]
Here is an example:

[array_shape_readme.pdf](https://github.com/stdlib-js/www/files/7260179/array_shape_readme.pdf)

We could consider having additional elements that are only visible when printing, e.g. some attribution / date of export. What do you think?

**Comment by Planeshifter** [comment]
Also a added a button to print the currently selected `README.md` per  a9495a12bc6b96f12a1a5f9536cebe695b20aba2.
# Dark theme:

![image](https://user-images.githubusercontent.com/1913638/135477033-c66713ed-8f74-4a22-b80a-30598f70b6d5.png)

# Light theme:

![image](https://user-images.githubusercontent.com/1913638/135477116-b5c52967-9f0d-4078-ac31-07967a1c41b3.png)

**Comment by kgryte** [comment]
Yeah, adding a copyright notice might be one thing to add, along with the date of export.

**Comment by Planeshifter** [comment]
And maybe the version of the docs for which the `README.md` was exported...

**Comment by kgryte** [comment]
Yeah, that makes sense, as well.
